planning/mission_planner/README.md

```diff
@@ -191,6 +191,30 @@ This interface for the MRM that pulls over the road shoulder. It has to be stopp
 This is a goal change to pull over, avoid parked vehicles, and so on by a planning component. If the modified goal is outside the calculated route, a reroute is required. This goal modification is executed by checking the local environment and path safety as the vehicle actually approaches the destination. And this modification is allowed for both normal_route and mrm_route.
 The new route generated here is sent to the AD API so that it can also be referenced by the application. Note, however, that the specifications here are subject to change in the future.
 
+#### Rerouting Static Obstacle
+
+`Rerouting Static Obstacle` is a node under `Mission Planner` that aims to change the planned route of the current already set route by finding an alternative route to the shortest planned one.
+The `Rerouting Static Obstacle` use-cases can be rerouting for a vehicle that is blocking the road, road construction that blocks some lanes, or any other similar scenarios of dynamic map information.
+
+The way that `Rerouting Static Obstacle` node is working currently is that, when the human driver/operator notices a blockage during the planned route of the mission, s/he points in the map (using a dedicated plugin in rviz) where the blockage is. Then `Rerouting Static Obstacle` searches for alternative routes and if successfully found it calls `change_route` API.
```

Comment:
I apologize very much for the very late review.
The change_route api will be removed as it has the same message type as set_route.

- https://github.com/autowarefoundation/autoware.universe/issues/6319
  - > `Internally change_route is merged into set_route.` 
- https://github.com/autowarefoundation/autoware.universe/pull/6363

This rerouting node needs to be updated as well.
We will need to coordinate whether to merge the PRs first for both. cc @isamu-takagi